config/brakeman.ignore

```diff
@@ -1,45 +1,46 @@
 {
   "ignored_warnings": [
     {
-      "warning_type": "Cross-Site Request Forgery",
-      "warning_code": 86,
-      "fingerprint": "9e450b4277235e592c083dc95e4902a07537932fa4f1bfb631d7b2438066564d",
-      "check_name": "ForgerySetting",
-      "message": "protect_from_forgery should be configured with 'with: :exception'",
-      "file": "app/controllers/application_controller.rb",
-      "line": 4,
-      "link": "https://brakemanscanner.org/docs/warning_types/cross-site_request_forgery/",
-      "code": "protect_from_forgery(:with => :null_session)",
+      "warning_type": "SQL Injection",
+      "warning_code": 0,
+      "fingerprint": "9b6c6ff75d840f01652e780a0f1f6e5605e42c01ec8446941eca5d1d426739b1",
+      "check_name": "SQL",
+      "message": "Possible SQL injection",
+      "file": "app/models/data_set.rb",
+      "line": 53,
+      "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
+      "code": "Arel.sql(\"location <-> #{\"'SRID=4326;POINT(#{location.longitude} #{location.latitude})'::geometry\"}\")",
       "render_path": null,
       "location": {
-        "type": "controller",
-        "controller": "ApplicationController"
+        "type": "method",
+        "class": "DataSet",
+        "method": "places_near"
       },
-      "user_input": null,
+      "user_input": "location.longitude",
       "confidence": "Medium",
-      "note": "This an API only so this is fine. We can't use the ActionController::API base class because we use various methods from the Base controller."
+      "note": "We can't use ? in Arel.sql, and sanitizing this string breaks the calls later on. But the location lat and lon always come from an RGeo object, so guaranteed to be numbers."
```

Comment:
Nice to see Brakeman actually being useful - I was wondering about this as I was reviewing that file (and came to the same conclusion), great to see it written out here so I don't have to double check!